native/ios/swift/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  SampleHandler.swift
+//  MeetHour_Broadcast_Extension
+//
+//  Created by Bharat Raj (VEMP) on 19/05/21.
+//  Copyright © 2021 Atlassian Inc. All rights reserved.
+//
```

Comment:
@chepury-bharathraj  - change created by and copyright o Meet Hour, LLC. Follow like other pages inherited from Broadcast extension.

Reply:
Sure Shoeb